common/reviews/api/node-core-library.api.md

```diff
@@ -587,10 +588,10 @@ export class Sort {
 export class StringBufferTerminalProvider implements ITerminalProvider {
     constructor(supportsColor?: boolean);
     readonly eolCharacter: string;
-    getErrorOutput(): string;
-    getOutput(): string;
-    getVerbose(): string;
-    getWarningOutput(): string;
+    getErrorOutput(normalizeColorCodes?: boolean): string;
```

Comment:
@iclanton Should this be an `options` parameter?  We might need to add more options later.  

And `getErrorOutput(true)` isn't very explicit about what `true` means.